gdk4/src/cairo_interaction.rs

```diff
@@ -45,7 +45,7 @@ pub trait GdkCairoContextExt {
 }
 
 impl GdkCairoContextExt for Context {
-    unsafe fn draw_from_gl(
+    fn draw_from_gl(
```

Comment:
This has to be unsafe. The `source` integer is a GL resource and if you pass random integers in there you'll have a lot of fun

Reply:
Aha. I've added a comment about the safety of the function. Not sure if the phrasing could be improved though